docs/core/entity.md

```diff
@@ -91,6 +89,42 @@ The following properties are used and controlled by Home Assistant, and should n
 | ------- | ------- | ------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | enabled | boolean | `True`  | Indicate if entity is enabled in the entity registry. It also returns `True` if the platform doesn't support the entity registry. Disabled entities will not be added to Home Assistant. |
 
+## Entity class attribute
+
+Writing property methods for each property is just a couple of lines of code,
+for example
+
+```python
+class MySwitch(SwitchEntity):
+
+    @property
+    def icon(self) -> str | None:
+        """Icon of the entity."""
+        return "mdi:door"
+
+    ...
+```
+
+The method returns a single, static, string in just a couple of lines code.
```

Comment:
I'd write this simpler without a long motivation.

Eg:
> Alternatively, a shorter form is to set Entity class attributes according to the following pattern:

> Example

Reply:
Agreed 👍